index.html

```diff
@@ -0,0 +1,190 @@
+    </ul>
+    <div class="work-button"><button type="button" >See Project</button></div>
+
+  </div>
+
+  <div class="work">
+    <img src="./images/imageplaceholder2.png" alt="Image placeholder">
+    <div class="title">Multi-Post Stories Gain+Glory</div>
+    <ul class="lang-box">
+        <li>html</li>
+        <li>javScript</li>
+        <li>css</li>
+        <li>Ruby on rails</li>
+    </ul>
+    <div class="work-button"><button type="button" >See Project</button></div>
+
+  </div>
+
+  <div class="work">
+    <img src="./images/imageplaceholder2.png" alt="Image placeholder">
+    <div class="title">Multi-Post Stories Gain+Glory</div>
+    <ul class="lang-box">
+        <li>html</li>
+        <li>javScript</li>
+        <li>css</li>
+        <li>Ruby on rails</li>
+    </ul>
+    <div class="work-button"><button type="button" >See Project</button></div>
+
+  </div>
+
+  <div class="work">
+    <img src="./images/imageplaceholder2.png" alt="Image placeholder">
+    <div class="title">Multi-Post Stories Gain+Glory</div>
+    <ul class="lang-box">
+        <li>html</li>
+        <li>javScript</li>
+        <li>css</li>
+        <li>Ruby on rails</li>
+    </ul>
+    <div class="work-button"><button type="button" >See Project</button></div>
+
+  </div>
+
+</div>
+
+</section>
+
+<section class="about-myself">
+<div class="heading-with-bg">
+    <h2 class="about-me-heading">About Me</h2>
+</div>
+
+    <p> I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+    don’t hesitate to contact me.</p>
+
+    <div class="about-me-button"><button type="button" class="resume-button" >Get My Resume</button></div>
+
+<div class="grid-container">
+
+    <div id="about-placeholder1" class="about-me">
+          <img class="prog-lang" src="./images/icon-languages.png" alt="Image placeholder">
+<h4 class="lang-text">Languages</h4>
+          <ul class="lang-box about">
+            <li>Html</li>
+            <li>JavaScript</li>
+            <li>CSS</li>
+            <li>Ruby</li>
+        </ul>
+    </div>
+
+    <div id="about-placeholder2" class="about-me">
+        <img class="prog-lang" src="./images/icon-frameworks.png" alt="Image placeholder">
+<h4 class="lang-text">Frameworks</h4>
+        <ul class="lang-box about">
+          <li>Bootstrap</li>
+          <li>Ruby on rails</li>
+          <li>RSpec</li>
+        </ul>
+        <ul class="lang-box about">
+          <li>Capybara</li>
+          <li>Selenium</li>
+      </ul>
+  </div>  
+
+    <div id="about-placeholder3" class="about-me">
+    <img class="prog-lang" src="./images/icon-skills.png" alt="Image placeholder">
+<h4 class="lang-text">Skills</h4>
+    <ul class="lang-box about">
+      <li>Codekit</li>
+      <li>GitHub</li>
+      <li>Codepen</li>
+      <li>Gitlab</li>
+    </ul>
+    <ul class="lang-box about">
+      <li>Terminal</li>
+    </ul>
+    </div>
```

Comment:
The designs look alike each time I inspect them in the browser. Nevertheless, I'll look into it.

Reply:
- [ ] Your design still seems to be having issues as you can see in the image below

![image](https://user-images.githubusercontent.com/90556221/165751936-a31ae24b-3664-4ac3-8460-e500278d2983.png)
The reason for this is because you have the entire section inside a class `grid-container` and your container currently has the following grid-template

![image](https://user-images.githubusercontent.com/90556221/165752196-396720ea-fe4f-4f72-b2ab-b2c152538f13.png)

As you can see you have 6 rows specified even though you have only 3 `about-placeholder`. You seem to be using this `grid-container` class in another location aswell so I'd suggest to create a new class `grid-container__about` and give it `grid-template-rows: repeat (3, 1fr)`. That will fix the issue :)